config/environments/production.rb

```diff
@@ -40,10 +43,18 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   # config.force_ssl = true
 
+  # Skip http-to-https redirect for the default health check endpoint.
+  # config.ssl_options = { redirect: { exclude: ->(request) { request.path == "/up" } } }
+
+  # Log to STDOUT by default
+  config.logger = ActiveSupport::Logger.new($stdout)
+    .tap  { |logger| logger.formatter = ::Logger::Formatter.new }
+    .then { |logger| ActiveSupport::TaggedLogging.new(logger) }
```

Comment:
Do we need to continue checking for the `RAILS_LOG_TO_STDOUT` env var?  I think that's what's been done in other apps.

Reply:
Done. I've also opened a Publishing API PR to do the same (I'd removed it in the update commit there): https://github.com/alphagov/publishing-api/pull/2854

Reply:
Thanks!  I hadn't spotted this in Publishing API.